modules/local/fusioninspector/environment.yml

```diff
@@ -2,9 +2,4 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - bioconda::dfam=3.7
-  - bioconda::hmmer=3.4
-  - bioconda::star-fusion=1.7.0
-  - bioconda::trinity=2.15.2
-  - bioconda::samtools=1.21
-  - bioconda::star=2.7.11b
```

Comment:
Fusioninspector now works without all of these dependencies?

Reply:
Probably not, this is part of the test to see what dependencies are necessary